src/schema/artwork/index.js

```diff
@@ -122,6 +122,88 @@ export const artworkFields = () => {
     category: {
       type: GraphQLString,
     },
+    attribution_class: {
+      type: GraphQLString,
+      description: "Attribution class as classification name",
+      resolve: artwork => {
+        if (artwork.attribution_class) {
+          return capitalizeFirstCharacter(artwork.attribution_class)
+        }
+      },
+    },
+    attribution_class_hover: {
```

Comment:
I'm thinking the naming here should maybe drop the `_hover` part (since that's a function of the current UI), and maybe be `attribution_class_info` or something?

Reply:
Agree, maybe `hint` so that it's not tied to a specific kind of device?

Reply:
👍